shells/web-shell/elements/web-planner.js

```diff
@@ -33,16 +33,16 @@ class WebPlanner extends Xen.Debug(Xen.Async, log) {
       log('planificator is disconnected and is disposing');
     }
     if (config && arc && !state.planificator) {
-      this.awaitState('planificator', async () => this._createPlanificator(config, arc, userid));
+      this.awaitState('planificator', async () => this._createPlanificator(config, arc));
     }
     if (state.planificator && search !== state.search) {
       state.search = search;
       state.planificator.setSearch(state.search);
     }
   }
-  async _createPlanificator(config, arc, userid) {
+  async _createPlanificator(config, arc) {
     const options = {
-      userid,
+      userid: 'user',
```

Comment:
Filed https://github.com/PolymerLabs/arcs/issues/3122 for planificator to stop relying on `userid`

Reply:
Good catch, thanks!